### DISCUSSION TITLE
Custom Fonts Support

### DISCUSSION THREAD

**Discussion by alanspike02** [Kas-tle/java2bedrock.sh#3366] [Feature Requests]
This converter has been very helpful. I don't know if this is possible, but if so, is it possible to convert the Java version of Custom Fonts as well?

**Comment by typicalsmc** [comment]
i would say it is possible but quite limited due to the difference between java and bedrock "custom font"

**Comment by Kas-tle** [comment]
It's possible but unlikely I will do this as the converter is already quite bloated for a bash script as it is, and as Typical correctly points out, the differences between bedrock and java fonts in terms of sizing make it difficult to convert on an automated basis. If this is something you wish to have in your pack, you should do so manually.
